test/channels.jl

```diff
@@ -198,6 +198,7 @@ for N in [0,10]
 end
 
 # Testing timedwait on multiple channels
+using Dates
```

Comment:
Will this cause problems if someone builds julia w/o the Dates module? I guess these tests would fail?

Reply:
Yes, but it is sort of the same if someone is building without Base64 though, right? I think for now, we don't have the requirement that arbitrary stdlib modules can be excluded.